internal/services/dispatcher/server.go

```diff
@@ -209,11 +209,11 @@ func (worker *subscribedWorker) CancelStepRun(
 	})
 }
 
-func (s *DispatcherImpl) Register(ctx context.Context, request *contracts.WorkerRegisterRequest) (*contracts.WorkerRegisterResponse, error) {
+func (d *DispatcherImpl) Register(ctx context.Context, request *contracts.WorkerRegisterRequest) (*contracts.WorkerRegisterResponse, error) {
```

Comment:
Drive-by, but the golint rules were complaining about using `s *DispatcherImpl` in some places and `d *DispatcherImpl` in other places, and I agree it makes the code harder to read, so changed all locations to use `d *DispatcherImpl`.